src/components/Course/CourseCaution.vue

```diff
@@ -44,7 +44,7 @@ const getCourseCautions = (course: FirestoreSemesterCourse): CourseCautions => {
   if (illegallyDoubleCountedCourseIDs.has(course.crseId)) {
     illegallyDoubleCountedCourseIDs.forEach(courseId => {
       doubleCountingRequirementWarning = requirementFulfillmentGraph
-        .getConnectedRequirementsFromCourse({ courseId, subject: '', number: '', credits: 0 })
+        .getConnectedRequirementsFromCourse({ courseId, uniqueId: -1, code: '', credits: 0 })
```

Comment:
Is the -1  for `uniqueId` intended to be a dummy value here?

Reply:
Yes